### ISSUE TITLE
Achei um bug em README.md

### ISSUE THREAD

**Issue by LeonardoGaldino** [ovictoraurelio/finger-tunes#1]
A resolução do bug fica como tarefa de casa para os desenvolvedores do projeto

**Comment by acmh** [comment]
@LeonardoGaldino I agree with your statement, please @ovictoraurelio correct what Leo says.

**Comment by ovictoraurelio** [comment]
This issue has no solution yet! New updates will be available in this topic soon.
